src/i18n/locales/de-DE.json

```diff
@@ -27,15 +27,16 @@
         "previewButton": "Vorschaumodus",
         "previewButton-tooltip": "Überspringt gelöschte Segmente bei der Wiedergabe des Videos. Derzeit {{status}}. Hotkey: {{hotkeyName}}",
         "previewButton-aria": "Vorschaumodus aktivieren oder deaktivieren. Hotkey: {{hotkeyName}}.",
-        "playButton-tooltip": "Play-Button",
-        "time-duration-tooltip": "Wiedergabezeit und -dauer",
+        "playButton-tooltip": "Video abspielen",
+        "pauseButton-tooltip": "Video pausieren",
+        "current-time-tooltip": "Aktuelle Zeit",
+        "time-duration-tooltip": "Videodauer",
```

Comment:
Updates to translations should only happen via Crowdin, this prevents merge conflicts from happening. Only the source language (`en-US`) should be altered in Pull Requests.